rllib/agents/slateq/slateq_torch_model.py

```diff
@@ -13,71 +13,64 @@
 
 
 class QValueModel(nn.Module):
-    """The Q-value model for SlateQ.
-
-    A simple MLP with n layers (w/ LeakyReLU activation), whose sizes are
-    specified via `config.q_hiddens`.
-    """
-
-    def __init__(self, embedding_size: int, q_hiddens: Sequence[int]):
+    def __init__(
+        self,
+        obs_space: gym.spaces.Space,
+        fcnet_hiddens_per_candidate=(256, 32),
+    ):
         """Initializes a QValueModel instance.
 
-        Args:
-            embedding_size: The sum of user- and doc embedding sizes. This is the input
-                dimension going into the first layer of the MLP.
-            q_hiddens: List of dense layer sizes to build the MLP by.
+        Each document candidate receives one full Q-value stack, defined by
+        `fcnet_hiddens_per_candidate`. The input to each of these Q-value stacks
+        is always {[user] concat [document[i]] for i in document_candidates}.
```

Comment:
Does q-value stack mean 1 q value per candidate document? That's what I've inferred from this statement.